localization/js/language-select.js

```diff
@@ -0,0 +1,47 @@
+import { CollisionComponent, Component, Property, TextComponent, Collider, Justification } from '@wonderlandengine/api';
+import { CursorTarget } from '@wonderlandengine/components';
+import { LanguageButton } from './language-button';
```

Comment:
`langauge-button.js`, always add the extension even if it works. That allows to be highly compatible with browser in case no bundling happens

Reply:
This obviously only is about relative path, not needed for imported libraries